lib/count_opened_loans/main.py

```diff
@@ -50,22 +49,28 @@ async def get_count_loan_approved_or_disbursed_events(
 
     alerts_by_institution_id = dict(zip(institution_ids, r))
 
-    alert_df_by_institution_id: Dict[str, pd.DataFrame] = {}
+    alerts_list_by_institution_id: Dict[str, List[Dict]] = {}
     for institution_id, alerts in alerts_by_institution_id.items():
         if alerts:
-            alert_df_by_institution_id[institution_id] = pd.DataFrame(
-                [alert.dict() for alert in alerts]
-            )
+            alerts_list_by_institution_id[institution_id] = [
+                alert.dict() for alert in alerts
+            ]
 
     count_loan_approved_or_disbursed_events = 0
-    for institution_id, alert_df in alert_df_by_institution_id.items():
 
-        time_window_filter = (alert_df.ts >= utc_starttime.timestamp()) & (
-            alert_df.ts < utc_endtime.timestamp()
-        )
-        alert_df_filtered = alert_df[time_window_filter]
+    for institution_id, alerts_list in alerts_list_by_institution_id.items():
+        filtered_alerts_list_on_time_window = [
+            alert_record
+            for alert_record in alerts_list
```

Comment:
Since we are using a standard for loop and not list compression here, should we update to standard for loops? It could also be easier if this is for others to move it to a different language.